integration-test/lincheck/build.gradle.kts

```diff
@@ -0,0 +1,30 @@
+sourceSets {
+    main {
+        java.srcDirs("src/main")
+    }
+
+    dependencies {
+        val junitVersion: String by project
+        val jctoolsVersion: String by project
+
+        implementation(project(":"))
+        implementation(rootProject.sourceSets["test"].output)
```

Comment:
Newer saw dependencies declared like this before, is it ok?
Anyway, what exactly it depends on?
Depending on the answer, we may consider moving it to `common`.